code/modules/projectiles/guns/ballistic/revolver.dm

```diff
@@ -44,7 +44,7 @@
 	recent_spin = world.time + spin_delay
 
 	if(do_spin())
-		playsound(usr, "revolver_spin", 30, FALSE)
+		playsound(usr, "revolver_spin", 100, FALSE)
```

Comment:
playsound(src

Reply:
Er... do you want me to put playsound(src, "revolver_spin", 100) instead?
Not a coder, if you can't tell.